README.md

```diff
@@ -7,6 +7,7 @@ A dockerized version of [AltServer-Linux](https://github.com/NyaMisty/AltServer-
 - Docker
 - Avahi running on host system
 - usbmuxd **NOT** running on host system
+- Device UDID (can be found using [this guide](https://discussions.apple.com/thread/250783627))
```

Comment:
Specify that this is for installations of the altstore apk, since this UDID is not really necessary for anyone to know or have if they have an existing installation of altstore which they just wish to refresh (which is afaik the most common user of this project)

Reply:
maybe i'd do it like `For installing the Altstore app if you don't already have it: Device UDID (can be found using [this guide](https://discussions.apple.com/thread/250783627))`

Reply:
I went ahead and moved it from the Requirements section to the Install on iOS section to keep the language clean and readable. Let me know what you think.

Reply:
you rock Gavin, thank you!!

Reply:
No problem! Happy to help!